### ISSUE TITLE
Salesforce Integration broken after every deploy

### ISSUE THREAD

**Issue by engram-design** [verbb/formie#1141]
### Discussed in https://github.com/verbb/formie/discussions/1140

<div type='discussions-op-text'>

<sup>Originally posted by **martinherweg** October 25, 2022</sup>
Hey,

i'm not sure if this is a bug so I thought I'll open a discussion first.
We have a formie installation with a salesforce integration currently in sandbox mode, this works fine until a new deployment happens, after that the integration says it's still connected but when I refresh the integration within a form I get a curl error.

So I debugged it locally and found out the the `apiDomain` variable within the `Salesforce.php` is null when this error happens, when I reconnect the integration the `apiDomain` variable is set and everything works fine.

Would anyone have a hint why it could break after deploying? Is there some kind of token mismatch happening or something like this? Or is it even something that is on the Salesforce side?

Thanks and regards,
Martin</div>

**Comment by engram-design** [comment]
@martinherweg If you're able to comment on what Formie/Craft version you're on?

**Comment by martinherweg** [comment]
@engram-design Sure!

PHP Version 8.0.21
Craft Version 4.2.8
Formie Version 2.0.13

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.14](https://github.com/verbb/formie/releases/tag/2.0.14)
